python/cudf/setup.py

```diff
@@ -21,8 +29,11 @@
             "../../cpp/include/cudf",
             "../../cpp/include",
             "../../cpp/build/include",
+            "../../thirdparty/cub",
+            "../../thirdparty/libcudacxx/include",
```

Comment:
Remove these after https://github.com/rapidsai/cudf/pull/3581 is merged.

Reply:
Why does removing bool8 allow you to remove these? We still use CUB and libcudacxx.

Reply:
`bool.hpp` was leaking third party include headers, which forces Cython to know about those headers. It'll be a solved problem once `bool.hpp` is removed.

Reply:
In general, Cython shouldn't need to know about CUB or libcudacxx.